### PR TITLE
Integration tests on CI

### PR DESCRIPTION
**Pull request by dtebbs**

### CODE REVIEW

File: scripts/ci [hunk 1]
```diff
@@ -44,18 +47,55 @@ function check_client() {
...
+function integration_tests() {
+    setup_client
+
+    # Use zeth scripts to start ganache (frm zeth root)
+    pushd depends/zeth
+    ganache_setup
+    ganache_start
+    popd
+
+    # Start aggregataor
```
Comment (AntoineRondelet) [inline scripts/ci:80]: aggregator

File: scripts/ci [hunk 2]
```diff
@@ -44,18 +47,55 @@ function check_client() {
...
+    . env/bin/activate
     make check
     deactivate
     popd
 }
 
+function integration_tests() {
+    setup_client
+
+    # Use zeth scripts to start ganache (frm zeth root)
```
Comment (AntoineRondelet) [inline scripts/ci:74]: from

Comment (dtebbs) [reply scripts/ci]: fixed typo.  thanks.

File: scripts/ci (hunk 1 above)
Comment (dtebbs) [reply scripts/ci]: fixed typo.  thanks.

**Review by AntoineRondelet — APPROVED** [review]
LGTM - thanks @dtebbs

**Pull request merged**